Source/Softswitch/Makefile

```diff
@@ -37,8 +37,15 @@ BIN_DIR := ../bin
 endif
 
 ifdef TRIVIAL_LOG_HANDLER
-SOFTSWITCH_DEPS = handler_log_t.h
-LH_CFLAGS = -D TRIVIAL_LOG_HANDLER
+LH_CFLAGS := $(LH_CFLAGS) -DTRIVIAL_LOG_HANDLER
```

Comment:
I feel like, since `TRIVIAL_LOG_HANDLER` and `SOFTSWITCH_INSTRUMENTATION` are enabled by default, we should have `DISABLE_` environment variables in `orchestrate-template.sh` as opposed to `ENABLE_` ones, so that we constrain the "scope" of the default to the lowest-reasonable place. :sweet_potato: Thoughts?

Reply:
Two questions here.
* should instrumentation be enabled by default? If not, then the current default state is temporary.

* Should the trivial log handler just "exist" unless a different log handler is specified?

Reply:
From discussion yesterday, the answer is yes to both (IIRC)

Reply:
Addressed the instrumentation part of this (setting it to be enabled by default and looking for a DISABLE_SOFTSWITCH_INSTRUMENTATION environment variable) in afc7d90